release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -228,9 +228,44 @@ For more information, see xref:../machine_management/control_plane_machine_manag
 [id="ocp-4-13-machine-config-operator"]
 === Machine Config Operator
 
+[id="ocp-4-13-machine-config-operator-layering"]
+==== Support for adding third party and custom content to {op-system}
+
+You can now use {op-system-first} image layering to add {op-system-base-full} and third-party packages to cluster nodes. 
+
+For more information, see  xref:../post_installation_configuration/coreos-layering.adoc#coreos-layering[{op-system-first} image layering].
+
+[id="ocp-4-13-machine-config-operator-core"]
+==== Support for setting the `core` user password
+
+You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). 
+
+For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#core-user-password_post-install-machine-configuration-tasks[Changing the core user password for node access].
+
 [id="ocp-4-13-nodes"]
 === Nodes
 
+[id="ocp-4-13-nodes-mirror"]
+==== Image registry repository mirroring by tags
+
+You can now pull images from a mirrored registry by using image tags in addition to digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` (ITMS) object to pull images by using image tags.
+
+If you have existing YAML files that you used to create ICSP objects, you can use the `oc adm migrate icsp` command to convert those files to an IDMS YAML file.
+
+For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
+
+For more information on converting existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
+
+[id="ocp-4-13-nodes-crun-ga"]
+==== crun general availability
```

Comment:
Does crun have to be in `` or pascal case?

Reply:
@snarayan-redhat As I understand, crun is a proper name that doesn't need mark-up and is always lower case. https://github.com/containers/crun